distributed/protocol/tests/test_serialize.py

```diff
@@ -192,7 +192,7 @@ def test_empty_loads_deep():
 
 
 def test_serialize_bytes():
-    for x in [1, "abc", np.arange(5)]:
+    for x in [1, "abc", np.arange(5), b"ab" * int(100e6)]:
```

Comment:
```suggestion
    for x in [1, "abc", np.arange(5), b"ab" * int(40e6)]:
```

Bringing this down to 80MB still causes the test to fail without the fix, and also reduces the testing time here from about 2s to 1s.

Reply:
(My apologies if shaving off 1s sounds trivial, but the test suite has thousands of these tests, so we try to be minimal when easy)

Reply:
Yes, I intentionally had it at 200MB to make sure we had more than 2 frames, but no objections in making it smaller, since it will still cover the issue we're fixing.

Reply:
> (My apologies if shaving off 1s sounds trivial, but the test suite has thousands of these tests, so we try to be minimal when easy)

Yes, I know, I also tend to reduce because of that, but I forgot to check how long it was for this one in particular.